presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -237,12 +237,14 @@ public void createPartitionedTable(Session session, HiveStorageFormat storageFor
                 ", _partition_smallint SMALLINT" +
                 ", _partition_integer INTEGER" +
                 ", _partition_bigint BIGINT" +
+                ", _partition_boolean BOOLEAN" +
                 ", _partition_decimal_short DECIMAL(3,2)" +
                 ", _partition_decimal_long DECIMAL(30,10)" +
+                ", _partition_date DATE" +
```

Comment:
I didn't include timestamp in the test because I ran into an unrelated issue where timestamp partitions were getting escaped twice (the ':' gets escaped and then the '%' from the escape sequence gets escaped), and only unescaped once, so then Presto can't parse the timestamp.  This occurs when doing a select * on the table without any filtering.  I can investigate the issue separately.